doc/design/wallet/wallet-design.md

```diff
@@ -320,7 +356,8 @@ pub extern fn indy_register_wallet_storage(command_handle: i32,
                                            add_record: Option<extern fn(storage_handle: i32,
                                                                         type_: *const c_char,
                                                                         id: *const c_char,
```

Comment:
I would like it more if id and type_ values are passed as pointer to buffer and len (not using base64 or similar). I agree there is small performance impact because these values are small enough (less than 100 bytes), but it still to me makes not a sense to do that.